tests/psychrometrics_test.py

```diff
@@ -1,12 +1,21 @@
 # coding=utf-8
 
 import unittest
-from ladybug.psychrometrics import find_saturated_vapor_pressure_torr, \
-    find_enthalpy, \
-    find_wet_bulb, \
-    find_dew_point, \
-    find_rel_humid_from_dry_bulb_dew_pt, \
-    find_air_temp_from_wet_bulb
+from ladybug.psychrometrics import \
```

Comment:
I feel `import ladybug.psychrometrics as psy` and then using `psy.function_name` will be cleaner but this will also work.

Reply:
Done!